ecosystem.json

```diff
@@ -1,18 +1,8 @@
 {
   "servers": [
-    {
-      "name": "Chainpoint-Roughtime",
-      "publicKeyType": "ed25519",
-      "publicKey": "bbT+RPS7zKX6w71ssPibzmwWqU9ffRV5oj2OresSmhE=",
-      "addresses": [
-        {
-          "protocol": "udp",
-          "address": "roughtime.chainpoint.org:2002"
-        }
-      ]
-    },
     {
         "name": "Cloudflare-Roughtime-2",
+        "version": "IETF-Roughtime",
```

Comment:
I guess we don't have a great way to indicate that this server supports both Google-Roughtime and IETF-Roughtime. This seems fine to me.

Reply:
Yeah that's a bit unfortunate. The best we can do is point to docs: https://github.com/cloudflare/roughtime/blob/4b43192f1fc0229069a249d498b87b6550cc7ec9/config/config.go#L31-L34

I suppose we could make this a list of versions instead of a single version?

Reply:
I think for now we can just assume that 'IETF-Roughtime' implies support for 'Google-Roughtime' as well,  and revisit later if we want to add any servers that support only the IETF version.